requirements.txt

```diff
@@ -7,5 +7,5 @@ django-debug-toolbar==3.4.0
 djangorestframework==3.13.1
 djangorestframework-simplejwt==5.2.0
 drf-spectacular==0.22.1
-Pillow==9.1.1
+Pillow
```

Comment:
Better to specify verion in requirements.txt

Reply:
There was an error with this module and never think that somebody will check requirements.txt))